app/controllers/Subscriptions.scala

```diff
@@ -63,11 +64,14 @@ class Subscriptions(
 
   def digital(countryCode: String): Action[AnyContent] = CachedAction() { implicit request =>
     implicit val settings: Settings = settingsProvider.settings()
-    val title = "Support the Guardian | Digital Subscription"
+    val title = "Support the Guardian | Digital Pack Subscription"
     val id = "digital-subscription-landing-page-" + countryCode
     val js = "digitalSubscriptionLandingPage.js"
     val css = "digitalSubscriptionLandingPageStyles.css"
-    Ok(views.html.main(title, id, js, css)).withSettingsSurrogateKey
+    val description = Some(stringsConfig.digitalPackLandingDescription)
+    val canonicalLink = Some(buildCanonicalDigitalSubscriptionLink(countryCode))
```

Comment:
In this instance `canonicalLink` doesn't need to be an `Option[String]` it can simply be of type `String` and you can remove the `Some` that wraps the function call. This will mean you won't have to check it's defined within your template.

Reply:
This is a parameter being sent to the main template for all pages, meaning it has to be optional or else all pages would need to incorporate it, right?